fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml

```diff
@@ -4,5 +4,5 @@ region: europe-west1
 description: Default subnet for dev Data Platform
 ip_cidr_range: 10.128.48.0/24
 secondary_ip_ranges:
-   pods: 100.64.48.0/20
+   pods: 100.65.16.0/20
    services: 100.64.64.0/24
```

Comment:
can you change services to 100.65 too?

Reply:
Ack Julio. 
Changing the service range in `fast/stages/2-networking-a-peering/data/subnets/dev/dev-gke-nodes-ew1.yaml1` to 100.65.64.0/24
Service range in `fast/stages/2-networking-a-peering/data/subnets/dev/dev-dataplatform-ew1.yaml` is 100.64.64.0/24 which is unchanged as these ranges do not overlap.